### PR TITLE
Adding tests folder exclusion

### PR DESCRIPTION
**Pull request by osesantos**

### CODE REVIEW

File: ViewGenerator/tools/webpack/Rules/Files.ts [hunk 1]
```diff
@@ -1,85 +1,91 @@
-﻿import { basename } from "path";
-import { RuleSetRule } from "webpack";
+﻿import {basename} from "path";
```
Comment (alvesmiguel1) [inline ViewGenerator/tools/webpack/Rules/Files.ts:1]: Actually, this file is only used for managing imports of files like images, GIFs, etc 😅

Comment (alvesmiguel1) [reply ViewGenerator/tools/webpack/Rules/Files.ts:1]: image

Comment (alvesmiguel1) [reply ViewGenerator/tools/webpack/Rules/Files.ts:1]: We should just update here, or use an "ignore" rule provided by webpack

**Review by alvesmiguel1 — CHANGES_REQUESTED** [review]

File: ViewGenerator/tools/webpack/Rules/Files.ts (hunk 1 above)
Comment (osesantos) [reply ViewGenerator/tools/webpack/Rules/Files.ts]: done :)

File: ViewGenerator/tools/webpack/Rules/TypeScript.ts [hunk 2]
```diff
@@ -9,7 +9,8 @@ import { getCurrentDirectory } from "../Plugins/Utils";
 const getTypeScriptRuleSet = (useCache: boolean): RuleSetRule => {
 
     const TypeScriptRule: RuleSetRule = {
-        test: /\.tsx?$/
+        test: /\.tsx?$/,
+        exclude: /tests/
```
Comment (alvesmiguel1) [inline ViewGenerator/tools/webpack/Rules/TypeScript.ts:13]: when doing tests in Studio, please confirm we are not doing type checking for tests also.
This file is intended for JS generation. Type checking is done separately by ForkTsCheckingPlugin or something like that... But it takes tsconfig.json as input, we have to exclude there

**Review by alvesmiguel1 — APPROVED** [review]

**Comment by alvesmiguel1** [comment]
don't forget to bump ViewGenerator

File: ViewGeneratorCore/tools/package-lock.json [hunk 3]
```diff
@@ -1,133 +1,168 @@
 {
+  "name": "tools",
```
Comment (alvesmiguel1) [inline ViewGeneratorCore/tools/package-lock.json:2]: hum... better revert this file

File: ViewGenerator/tools/webpack/Rules/TypeScript.ts (hunk 2 above)
Comment (osesantos) [reply ViewGenerator/tools/webpack/Rules/TypeScript.ts:13]: thats great advice! thanks a lot. I will look at that :D

**Review by mpm-os — APPROVED** [review]

**Pull request merged**